### ISSUE TITLE
Suspicious token split

### ISSUE THREAD

**Issue by simon-clematide** [UniversalDependencies/UD_German#10]
The split of "am" in the third word seems to be a conversion error/artefact.  There is no way that am could be split twice.
```
# sent_id = train-s7888
# text = Es wurde am Mariinski - Theater in Sankt Petersburg mit der Choreografie von Marius Petipa uraufgeführt.
1	Es	es	PRON	PPER	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	19	nsubj:pass	_	_
2	wurde	werden	AUX	VAFIN	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	19	aux:pass	_	_
3-6	am	_	_	_	_	_	_	_	_
3	an	an	ADP	APPR	_	19	dep	_	_
4	dem	der	DET	ART	Definite=Def|PronType=Art	19	det	_	_
5	an	an	ADP	APPR	_	7	case	_	_
6	dem	der	DET	ART	Case=Dat|Definite=Def|Gender=Masc,Neut|Number=Sing|PronType=Art	7	det	_	_
7	Mariinski	Mariinski	PROPN	NN	_	19	obl	_	_
8	-	-	PUNCT	$(	_	7	punct	_	_
9	Theater	Theater	PROPN	NN	Case=Dat|Gender=Masc,Neut|Number=Sing	7	flat	_	_
10	in	in	ADP	APPR	_	11	case	_	_
11	Sankt	Sankt	PROPN	NE	_	7	nmod	_	_
12	Petersburg	Petersburg	PROPN	NE	_	11	flat	_	_
13	mit	mit	ADP	APPR	_	15	case	_	_
14	der	der	DET	ART	Case=Dat|Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	_	_
15	Choreografie	Choreografie	NOUN	NN	Case=Dat|Gender=Fem|Number=Sing	19	obl	_	_
16	von	von	ADP	APPR	_	17	case	_	_
17	Marius	Marius	PROPN	NE	_	15	nmod	_	_
18	Petipa	Petipa	PROPN	NE	Case=Dat|Number=Sing	17	flat	_	_
19	uraufgeführt	uraufgeführt	VERB	ADJD	_	0	root	_	SpaceAfter=No
20	.	.	PUNCT	$.	_	19	punct	_	_

```

**Comment by dan-zeman** [comment]
Komisch...

Fixed in b83fa2f. I hope it is an isolated case.

**Comment by simon-clematide** [comment]
It is. My conversion script stumbled over it only once.
